public/static/app/perf-bb/SignalProcessingCtrl.test.js

```diff
@@ -69,6 +69,7 @@ describe('SignalProcessingFactoriesTest', () => {
           create_time: ">now",
           project: "=my-project",
           probability: ">3.14",
+          magnitude: ">=.005"
```

Comment:
The default should be 0.05. Since this is used in 2 places, can we extract a constant?